cumulus/polkadot-parachain/src/command.rs

```diff
@@ -253,6 +260,22 @@ fn load_spec(id: &str) -> std::result::Result<Box<dyn ChainSpec>, String> {
 			para_id.expect("Must specify parachain id"),
 			"polkadot-local",
 		)),
+		"penpal-westend" => Box::new(chain_spec::penpal::get_penpal_chain_spec(
+			para_id.expect("Must specify parachain id"),
+			"westend-local",
+		)),
```

Comment:
How did this end up in here?

Reply:
I got carried away originally and was going to create it and missed this in the revert.